### ISSUE TITLE
Search?

### ISSUE THREAD

**Issue by Haroenv** [Raathigesh/hooks.guide#4]
It would be nice to use this centralised place to be able to search across all docs of the hooks. 

We can add DocSearch (like on the react docs) on this page. What do you think?

**Comment by Raathigesh** [comment]
That would be lovely to have. I just applied for it. Thank you for suggesting it 👍 💯

**Comment by vivek12345** [comment]
@Raathigesh Have you started working on this?

**Comment by Raathigesh** [comment]
I applied through https://community.algolia.com/docsearch/. Apparently, there will be details sent to my mail when they review the request. I'll update here when I hear back from them :)

**Comment by Haroenv** [comment]
Yes, a colleague of mine will get back to you in a few hours. You can create a React DocSearch component like this one:

https://github.com/facebook/immutable-js/blob/master/pages/src/docs/src/DocSearch.js

or this one:

https://github.com/reactjs/reactjs.org/blob/master/src/components/LayoutHeader/DocSearch.js

**Comment by vivek12345** [comment]
Thanks for reaching out @s-pace. I will start reading on the selector format that you mentioned and will accordingly make the changes. 👍

**Comment by Raathigesh** [comment]
Hey @s-pace,

Sorry about the delay. I have added classes prefixed with "doc-" to the navigation items as well as the content items. Please let me know if that helps or anything else we need to do.
